openapi/openapi.yaml

```
@@ -49,381 +57,612 @@ components:
   - name: Status
-    description: |
-      Check the status of the Rebilly API (no authentication required).
+    description: >-
+      Check the status of the Rebilly API. No authentication is
+      required.
   - name: Tracking
-    description: |
-      Tracking is a layer for accessing all the activity (API requests,
-      subscriptions, webhooks, events, etc.), thus providing easier
-      debugging and issues auditing.
+    description: >-
+      Tracking is a layer for accessing all activity such as: API
+      requests, subscriptions, webhooks, events, and more. Use
+      tracking when debugging and auditing issues.
   - name: Users
-    description: |
-      A User represents a person who can login to Rebilly, and take actions subject to
-      their granted permissions.
+    description: >-
+      A user is a person who can login to Rebilly, and take actions
+      based on their granted permissions.
   - name: Websites
-    description: |
-      A Website represents the website/brand that customers interact with... You
-      could think of it like a brand.  For example, Nestle owns Perrier and Purina
-      and PowerBar.
+    description: |-
+      A website is where your organization obtains a customer. It is
+      also the processor account and billing descriptor used for
+      payment transactions. You can create multiple websites.
 
-      We recognize that some enterprises have more than one website (or brand).  The
-      website is related to each invoice and each payment gateway account.  This feature
-      would allow you to have gateway accounts that are related to multiple websites, or
-      exclusive to particular websites.  And gives you more control over your business.
-  - name: Webhook Credentials
-    description: |
-      Get and create Webhook credential hashes.
+      Websites are related to each invoice and each payment
+      gateway account. This enables you to associate gateway accounts
+      with multiple websites, or make them exclusive to particular
+      websites. For more information, see
+      [My organizations and websites](https://www.rebilly.com/docs/settings/organizations-and-websites/).
+  - name: Webhook credentials
+    description: >-
+      Use these operations to manage webhook credential hashes.
   - name: Webhooks
-    description: |
-      Webhooks are designed to notify your systems when certain/all registered events happen in near real-time.
-      They allow you to collect information about those events. Rebilly can send these information via
-      POST to an URL of your choice.
+    description: |-
+      Use these operations to manage webhooks. Webhooks notify your
+      systems in real-time when certain events occur. For example,
+      when a new transaction occurs or a new subscription is created.
+      Webhooks enable you to collect information about events. Rebilly
+      sends this information by HTTP POST request to a defined URL of
```

```suggestion
      can send this information by HTTP POST request to a defined URL of
```

HTTP verb can be configured for a werbhook: GET, POST, PUT, PATCH, DELETE

---

Added in [2a1db2a](https://github.com/Rebilly/api-definitions/pull/966/commits/2a1db2a918553b0aec0eb1840803d65994a23c92)